YAKL_atomics.h

```diff
@@ -195,6 +195,40 @@
     __device__ __forceinline__ void atomicMax(unsigned long long int &update , unsigned long long int value) {
       ::atomicMax( &update , value );
     }
+  #endif
+
+  #ifdef __USE_OPENMP45__
+    template <class T> inline void atomicAdd(T &update, T value) {
+      #pragma omp atomic update 
+        update += value;
+    }
+    template <class T> inline void atomicMin(T&update, T value) {
+      #pragma omp critical
+      {
+        if (value < update){update = value;}
```

Comment:
Please change this to `update = value < update ? value : update`

Same for the max atomic.

Reply:
Updated but still fails atomic test.